libraries/push/impl/src/main/kotlin/io/element/android/libraries/push/impl/notifications/NotifiableEventResolver.kt

```diff
@@ -83,6 +83,11 @@ class NotifiableEventResolver @Inject constructor(
         return when (val content = this.content) {
             is NotificationContent.MessageLike.RoomMessage -> {
                 val messageBody = descriptionFromMessageContent(content, senderDisplayName ?: content.senderId.value)
+                val notificationBody = if (hasMention) {
+                    stringProvider.getString(R.string.notification_mentioned_you_body, messageBody)
```

Comment:
The value of the string is:
```xml
  <string name="notification_mentioned_you_body">"%1$s mentioned you.
%2$s"</string>
```
so it seems like a parameter is missing here? The `messageBody` will be used for `%1$s` and it should be the sender. Can you double check please?

Reply:
Maybe you want to use `notification_mentioned_you_fallback_body` instead?

Reply:
It's not anymore: https://github.com/vector-im/element-x-android/pull/1846/files#diff-ceb20f078550a897a919802d4b7635a8299f6ac1ef679aefba72e4c294954f2dR11.

`notification_mentioned_you_fallback_body` was removed for this same reason. We decided to change what the message looked like since it was too verbose.

Reply:
OK, it's fine now. I do not know what happened with all those string values, and it does not worth investigate more about it. Thanks!